doc/files.md

```diff
@@ -6,13 +6,13 @@
 * blocks/rev000??.dat; block undo data (custom); since 0.8.0 (format changed since pre-0.8)
 * blocks/index/*; block index (LevelDB); since 0.8.0
 * chainstate/*; block chain state database (LevelDB); since 0.8.0
-* database/*: BDB database environment; only used for wallet since 0.8.0
-* db.log: wallet database log file
```

Comment:
Follow `blktree/` and `blocks/index/*` example:
```
* db.log: wallet database log file; replaced by wallets/ directory in 0.16.0
* wallets/db.log; since 0.16.0
```
Same for `wallet.dat`.